File: easy/e9/python3/solution.py
class Solution:
    def isPalindrome(self, x: int) -> bool:
        if x < 0:
            return False

        origX = x
        n = 0

        while x > 0:
            n = (n * 10) + (x % 10)

            x = x // 10

        if origX == n:
            return True
        else:
            return False

File: easy/e9/typescript/solution.ts
export function isPalindrome(x: number): boolean {
    if (x < 0) {
        return false;
    }

    let origX = x;
    let result: number = 0;

    for (let i: number = 1; 0 < x; i = i * 10) {
        result = result * 10 + Math.trunc(x % 10);

        x = Math.trunc(x / 10);
    }

    if (result != origX) {
        return false;
    }

    return true;
};